src/NuGet.Core/NuGet.Frameworks/NuGetFrameworkFactory.cs

```diff
@@ -508,21 +508,21 @@ private static bool TryParseCommonFramework(string frameworkString, out NuGetFra
                 case "netstandard21":
                     framework = FrameworkConstants.CommonFrameworks.NetStandard21;
                     break;
-                case "netstandardapp1.5":
-                case "netstandardapp15":
-                    framework = FrameworkConstants.CommonFrameworks.NetStandardApp15;
+                case "netcoreapp2.1":
+                case "netcoreapp21":
+                    framework = FrameworkConstants.CommonFrameworks.NetCoreApp21;
                     break;
-                case "netcoreapp1.0":
-                case "netcoreapp10":
-                    framework = FrameworkConstants.CommonFrameworks.NetCoreApp10;
+                case "netcoreapp3.1":
+                case "netcoreapp31":
+                    framework = FrameworkConstants.CommonFrameworks.NetCoreApp31;
                     break;
-                case "netcoreapp2.0":
-                case "netcoreapp20":
-                    framework = FrameworkConstants.CommonFrameworks.NetCoreApp20;
+                case "netcoreapp5.0":
```

Comment:
I'd just have all of these point to the same framework constant. 
No need to declare an extra static.

Reply:
let me examine and think through the netcoreapp5.0 deprecation (or whatever) step post-preview2.

Reply:
fixed in next commit. will approach the netcoreapp5.0 behavior for post-preview2 seperately. (design and implementation)